RabbitMQ.Stream.Client/HeartBeatHandler.cs

```diff
@@ -62,15 +67,15 @@ private async Task PerformHeartBeatAsync()
 
         // missed the Heartbeat 
         Interlocked.Increment(ref _missedHeartbeat);
-        LogEventSource.Log.LogWarning($"Heartbeat missed: {_missedHeartbeat}");
+        _logger?.LogWarning($"Heartbeat missed: {_missedHeartbeat}");
```

Comment:
This should be `_logger.LogWaring("Heartbeat missed {MissedHearBeatCounter}", _missedHeartbeat)`. Logger caches the template that is provided so by interpolating we pollute the cache.

Reply:
Feel free to make changes!